src/middlewares/verifyToken.js

```diff
@@ -27,18 +27,24 @@ export default (request, response, next) => {
 
   jwt.verify(token, secret, async (error, decoded) => {
     if (error) {
-      const message = (error.name === 'TokenExpiredError') ? 'token expired, you have to be signed in to continue' : 'you have to be signed in to continue';
-      return responseMessage(response, 401, { error: message });
+      const message = (error.name === 'TokenExpiredError') ? 'token expired' : 'invalid token';
+      responseMessage(response, 401, { error: message });
     }
     try {
+      const blacklistedToken = await findBlacklistedToken(token);
+      if (blacklistedToken) {
+        return responseMessage(response, 401, { error: 'invalid token' });
+      }
       const user = await findUser(decoded.id);
```

Comment:
Do you think `const blacklistedToken = await findBlacklistedToken(token);` and `const user = await findUser(decoded.id);` should be in the same try{} catch{} block?

Reply:
From my research, using one try/catch block containing multiple await operations is okay. I also learned that if an error should occur in any of the operations, It gets caught in the catch block. But I'm learning every day and I would be happy to hear any suggestions for a better way to do it.